src/vs/platform/quickinput/browser/quickInput.ts

```diff
@@ -745,20 +745,11 @@ export class QuickPick<T extends IQuickPickItem> extends QuickInput implements I
 		return this._customButtonLabel;
 	}
 
-	set customLabel(label: string | undefined) {
+	set customLabel(label: string | IMarkdownString | undefined) {
```

Comment:
can you try using this instead:
https://github.com/microsoft/vscode/blob/44ffc28fea5da56f1e447939da3621d644fe86ce/src/vs/base/browser/ui/button/button.ts#L26

then the API doesn't have to change at all